api/api.yaml

```diff
@@ -708,6 +753,20 @@ components:
           $ref: '#/components/schemas/ObjectID'
         name:
           type: string
+    InboxInfo:
+      required:
+        - messageCount
+      properties:
+        messageCount:
+          description: Number of new messages in the inbox.
```

Comment:
What is a new message? Can there also be an old message? If so, does the name `messageCount` not suggest to count all, old and new messages?